### PR TITLE
VideoPress: Add loading state at `AdminPage`

### LINKED ISSUE #26170: VideoPress: implement loading state for the Grid view
https://user-images.githubusercontent.com/77539/192356601-c15e0ea2-bc24-4aea-8348-7c9410bd53cc.mov

### PR DESCRIPTION
**Pull request by renatoagds**
Add loading state at `VideoCard` component and use it on dashboard and delete video.

⚠️ Missing support for loading in `VideoRow`

Closes #26170 

#### Changes proposed in this Pull Request:

* Add new `loading` prop at `VideoCard` and `VideoGrid`.
* Update `VideoThumbnail` to support placeholder usage.
* Fix a problem where `editable` was not being rendered.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Start `VideoPress`
* Navigate to `Dashboard`.
* You should see loading state.

### CODE REVIEW

File: projects/packages/videopress/src/client/admin/components/video-grid/index.tsx [hunk 1]
```diff
@@ -30,26 +30,28 @@ const getThumbnail = ( { video }: { video: VideoPressVideo } ): React.ReactNode
...
 	};
+
 	return (
 		<div className={ styles.wrapper }>
 			<Container fluid horizontalSpacing={ 0 } horizontalGap={ 0 }>
 				{ gridVideos.map( ( video, index ) => {
 					return (
 						<Col key={ index } sm={ 4 } md={ 4 } lg={ 4 }>
-							<ConnectVideoCard
+							<VideoCard
```
Comment (retrofox) [inline projects/packages/videopress/src/client/admin/components/video-grid/index.tsx:46]: This change should be at least discussed. There is an intentionality in creating the Connect component.

Comment (retrofox) [reply projects/packages/videopress/src/client/admin/components/video-grid/index.tsx:46]: The idea behind this component is to delete the task to pull the data independently of the context where the card renders.

It only requires the video ID. In this way, the component can handle global states from its PoV (like when the app is loading all videos which this case, as well as individual states such as when the video is being edited, added, or removed).

Why are you removing this implementation?

**Review by retrofox — CHANGES_REQUESTED** [review]
Renato, left a comment about removing the Connect component. Can you consider don't get rid of it? Thanks.

**Comment by nunyvega** [comment]
About the testing, this is how it looks:

https://user-images.githubusercontent.com/16329583/193247855-7d7f6192-9a46-461c-b135-7e9a53b4f15c.mov



When I load the page it works as expected. When I delete the video, it takes a bit and then changes to the loading state.

**Comment by retrofox** [comment]
> When I load the page it works as expected. When I delete the video, it takes a bit and then changes to the loading state.

This is because the component that takes over detecting and changing the UI is the Connect one (removed in this PR). The state registers when the video is being removed and when it was successfully removed, to reflex it to the UI.

File: projects/packages/videopress/src/client/admin/components/video-grid/index.tsx (hunk 1 above)
Comment (renatoagds) [reply projects/packages/videopress/src/client/admin/components/video-grid/index.tsx:46]: @retrofox I’m not removing it. It’s only the default export now.

**Comment by retrofox** [comment]
There is a regression when removing the video. It doesn't change the UI. something that we can handle in a follow-up PR.

**Review by retrofox — APPROVED** [review]
LGTM. The UI doesn't reflex when removing the video. It currently shows the blank state of the VideoCard, but it can be addressed a follow-up https://github.com/Automattic/jetpack/pull/26555

**Pull request merged**